Mathlib/MeasureTheory/Constructions/Pi.lean

```diff
@@ -780,7 +780,7 @@ theorem volume_measurePreserving_sumPiEquivProdPi_symm (π : ι ⊕ ι' → Type
     MeasurePreserving (MeasurableEquiv.sumPiEquivProdPi π).symm volume volume :=
   measurePreserving_sumPiEquivProdPi_symm (fun _ ↦ volume)
 
-theorem measurePreserving_sumPiEquivProdPi {π : ι ⊕ ι' → Type*} [∀ i, MeasurableSpace (π i)]
+theorem measurePreserving_sumPiEquivProdPi {π : ι ⊕ ι' → Type*} {_m : ∀ i, MeasurableSpace (π i)}
```

Comment:
I think this change to implicit arguments is worth at least one of:
* A mention in the PR description
* A library note
* A change to core to make this not necessary in the first place 🙃

Reply:
I added it to the PR description.
I believe this is done to work with multiple `MeasurableSpace` instances on the same type. 
I personally don't care about this, and this change is not necessary for me, but I wanted to uniformize it.

Reply:
Maybe worth mentioning that the motivation was "for consistency with other lemmas"